scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: Command Injection

The `executeCommand` function is vulnerable to command injection attacks[^2]. Unsanitized user input is directly used in a shell command, which can lead to arbitrary command execution.

Recommendation: Avoid using user input directly in shell commands. If necessary, use a whitelist of allowed commands or parameters, and properly sanitize and validate all user inputs.
```suggestion
function executeCommand(userInput: string) {
    // Use a whitelist of allowed commands
    const allowedCommands = ['ls', 'dir'];
    if (allowedCommands.includes(userInput)) {
        child_process.exec(userInput);
    } else {
        throw new Error('Invalid command');
    }
}
```